docs/shared-dependencies.md

```diff
@@ -0,0 +1,118 @@
+# Proposal: Shared Dependencies
+
+_Note, this is currently a proposal. Once the proposal has been turned into
+an implementation this can be updated to document the implementation instead
+of being a proposal._
+
+Share dependencies are those that more than one chart may share. These are
+useful for software running as system services.
+
+## Use Case
+
+As someone installing an application, I would like to install shared dependencies
+if they are not present.
+
+### Prometheus Example
+
+Consider the case where cluster operators want to have a single instance of
+Prometheus running as a system wide service. There should only be one instance
+running in the cluster and all (or most) applications that need prometheus
+would use this single instance.
+
+When someone installs an application (e.g., another system service) that depends
+on Prometheus then Hypper should check to see if Prometheus is installed in the
+expected location. If not, it should attempt to install it. If Prometheus is
```

Comment:
How would Hypper install a service as a 'system wide' service ?

Reply:
Think of this as a cluster admin needing to install something just once at the cluster level. Like an operator, logging, etc.